geojson/src/commonMain/kotlin/org/maplibre/spatialk/geojson/BoundingBox.kt

```diff
@@ -117,11 +117,11 @@ public class BoundingBox(public val coordinates: DoubleArray) {
 
     public companion object {
         @JvmStatic
-        public fun fromJson(json: String): BoundingBox =
+        public fun fromJson(@Language("json") json: String): BoundingBox =
```

Comment:
these make IntelliJ automatically syntax highlight json string literals passed to these methods